.github/workflows/docker-build-and-publish.yml

```diff
@@ -70,7 +70,7 @@ jobs:
               GHCR_NAMESPACE="${{ github.repository_owner }}/gow"
             fi
             DOCKERHUB_NAMESPACE="${{ env.DOCKERHUB_NAMESPACE }}"
-            ARM64_RUNNER=${{ env.ARM64_RUNNER != '' && env.ARM64_RUNNER || 'ubuntu-latest' }}
+            ARM64_RUNNER=${{ env.ARM64_RUNNER != '' && env.ARM64_RUNNER || 'ubuntu-24.04-arm' }}
```

Comment:
BTW, GitHub provides public runners for arm64 since January (https://github.blog/changelog/2025-01-16-linux-arm64-hosted-runners-now-available-for-free-in-public-repositories-public-preview/)

I tested it on my repo and its pretty much as fast as the amd64 runner!